custom/icds_reports/models/util.py

```diff
@@ -69,3 +72,74 @@ class AggregationRecord(models.Model):
+            ('static-usage_forms', 'xmlns', 'http://openrosa.org/formdesigner/376FA2E1-6FD1-4C9E-ACB4-E046038CD5E2'),
+            ('static-usage_forms', 'xmlns', 'http://openrosa.org/formdesigner/D4A7ABD2-A7B8-431B-A88B-38245173B0AE'),
+            ('static-usage_forms', 'xmlns', 'http://openrosa.org/formdesigner/89097FB1-6C08-48BA-95B2-67BCF0C5091D'),
+            ('static-usage_forms', 'xmlns', 'http://openrosa.org/formdesigner/792DAF2B-E117-424A-A673-34E1513ABD88'),
+            ('static-usage_forms', 'xmlns', 'http://openrosa.org/formdesigner/619B942A-362E-43DE-8650-ED37026D9AC4'),
+            ('static-usage_forms', 'xmlns', 'http://openrosa.org/formdesigner/1D568275-1D19-46DB-8C54-2C9765DF6335'),
+            ('static-usage_forms', 'xmlns', 'http://openrosa.org/formdesigner/362f76b242d0cfdcec66776f9586dc3620e9cce5'),
+            ('static-usage_forms', 'xmlns', 'http://openrosa.org/formdesigner/756ec44475658f3f463f8012632def2bc9fbe731'),
+            ('static-dashboard_growth_monitoring_forms', 'xmlns', 'http://openrosa.org/formdesigner/7a55754119359466b1951d7251068bd4f45e73c3'),
+            ('static-awc_mgt_forms', 'xmlns', 'http://openrosa.org/formdesigner/D8EED5E3-88CD-430E-984F-45F14E76A551'),
+            ('static-cbe_form', 'xmlns', 'http://openrosa.org/formdesigner/61238C23-7059-446D-8A9C-34107642CBB2'),
+            ('static-cbe_form', 'xmlns', 'http://openrosa.org/formdesigner/D305345E-94AE-4A23-899E-22D05EECF1AD'),
+            ('static-infrastructure_form_v2', 'xmlns', 'http://openrosa.org/formdesigner/BEB94AFD-E063-46CC-AA75-BECD3C0FC20C'),
+            ('static-it_report_follow_issue', 'xmlns', 'http://openrosa.org/formdesigner/0AD845EA-69E8-4479-9140-4072A14AA0E5'),
+            ('static-it_report_follow_issue', 'xmlns', 'http://openrosa.org/formdesigner/32083FFC-9AD7-46A8-B256-1B9431469262'),
+            ('static-ls_home_visit_forms_filled', 'xmlns', 'http://openrosa.org/formdesigner/327e11f3c04dfc0a7fea9ee57d7bb7be83475309'),
+            ('static-ls_vhnd_form', 'xmlns', 'http://openrosa.org/formdesigner/b8273b657bb097eb6ba822663b7191ff6bc276ff'),
+            ('static-vhnd_form', 'xmlns', 'http://openrosa.org/formdesigner/A1C9EF1B-8B42-43AB-BA81-9484DB9D8293'),
+            ('static-visitorbook_forms', 'xmlns', 'http://openrosa.org/formdesigner/08583F46-ED60-4864-B54F-CA725D5C230E'),
+            ('static-commcare_user_cases', 'case_type', 'commcare-user'),
+            ('static-ccs_record_cases', 'case_type', 'ccs_record'),
+            ('static-child_health_cases', 'case_type', 'child_health'),
+            ('static-hardware_cases', 'case_type', 'hardware'),
+            ('static-household_cases', 'case_type', 'household'),
+            ('static-person_cases_v3', 'case_type', 'person'),
+            ('static-tasks_cases', 'case_type', 'tasks'),
+            ('static-tech_issue_cases', 'case_type', 'tech_issue'),
+        )
```

Comment:
This could have been generalized by loading all UCRs and parsing the case type or xmlns from the configured filter, but for now I think this is sufficient. 

An assumption/optimization here is that if it made it into one UCR it should have made it into all of them (for example growth monitoring forms (xmlns = http://openrosa.org/formdesigner/b183124a25f2a0ceab266e4564d3526199ac4d75) is in usage_forms and a couple of other UCRs, but its only checked with usage_forms. I did this since it should be much faster as it would reuse the usage_forms doc_id index instead of trying to query many large tables